client-go/controller/client/client_test.go

```diff
@@ -8,7 +8,7 @@ import (
 	"testing"
 )
 
-const sFile string = `{"username":"t","ssl_verify":false,"controller":"http://d.t","token":"a"}`
+const sFile string = `{"username":"t","ssl_verify":false,"controller":"http://d.t","token":"a", "response_limit": 50}`
```

Comment:
Minor, but could you omit the spaces in this JSON string? It just reads strangely to have inconsistent spacing here.